src/aem/azext_aem/azext_metadata.json

```diff
@@ -1,3 +1,3 @@
 {
-    "azext.minCliCoreVersion": "2.0.30"     
+    "azext.minCliCoreVersion": "2.19.1"     
```

Comment:
How do you know what the minimum cli core version should be? I need to upgrade it for the vmware extension.

Reply:
normally unless the extension depends on a specific change in core, no need to bump this version. Otherwise it will require user to install latest cli core before using the extension

Reply:
fyi. I ended up testing vmware extension in the docker containers and settled on `2.11.0` as the minimum version for it. #3045 #3046

Reply:
The aem extension requires a fairly new Azure Python SDK. I tested it with 2.17 and it did not work. So I upgraded to the latest one (2.19.1) and it worked so that is why I require this version.